src/expand_lti_launch.js

```diff
@@ -5,6 +5,10 @@ export default function handleTrayExpand(event) {
     iframe => iframe.contentWindow === event.source
   );
   if (sender) {
-    sender.closest('[role=dialog]').parentElement.style.width = '900px';
+    const dialog = sender.closest('[role=dialog]');
+    
+    if (dialog.getAttribute('aria-label') === 'Atomic Author') {
+      dialog.parentElement.style.width = 'width: 888px; max-width: 100%;';
```

Comment:
Discussed with Branden. He wants to use 888px as a width when the screen size is larger than 888px. Otherwise, the content should fill the entire screen

Reply:
This code doesn't work, since it's assigning to the 'width' attribute. Something like this would work

```
      dialog.parentElement.style.width = '888px';
      dialog.parentElement.style.maxWidth = '100%';
```